tests/Rule/RuleTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabberworm\CSS\Tests\Value;
```

Comment:
Please let's move this to `Tests/Unit/` (so I won't have to when I move/migrate the other existing tests).

Reply:
Done and repushed.